providers/os/provider/provider.go

```diff
@@ -419,10 +419,17 @@ func (s *Service) connect(req *plugin.ConnectReq, callback plugin.ProviderCallba
 		if err != nil {
 			return nil, err
 		}
-		fingerprint, err := IdentifyPlatform(conn, asset.Platform, []string{ids.IdDetector_Hostname})
-		if err == nil {
-			asset.Name = fingerprint.Name
-			asset.PlatformIds = fingerprint.PlatformIDs
+		// This is a workaournd to set Google COS platfomr IDs when scanned from inside k8s
```

Comment:
```suggestion
		// This is a workaournd to set Google COS platform IDs when scanned from inside k8s
```

Reply:
Oh, there are even more typos in this sentence... I'll get a PR up for this.

Reply:
https://github.com/mondoohq/cnquery/pull/2464/files